jupyterlab_caip_optimizer/README.md

```diff
@@ -0,0 +1,61 @@
+# Google Cloud Platform AI Optimizer Extension
```

Comment:
A lot of these instructions are no longer applicable (probably still need to update part of the Cookie Cutter project). I would suggest trimming it down to include only the things pertinent to your project similar to https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_gcpscheduler/README.md.